src/components/VideoChatButtonAndMenu/BaseVideoChatButtonAndMenu.js

```diff
@@ -40,15 +41,15 @@ function BaseVideoChatButtonAndMenu(props) {
             text: props.translate('videoChatButtonAndMenu.zoom'),
             onPress: () => {
                 setIsVideoChatMenuActive(false);
-                Linking.openURL(CONST.NEW_ZOOM_MEETING_URL);
```

Comment:
@sobitneupane @Nodebrute I can still see some usage of this method in App repo, do you think it makes sense to update them(at least for external links) otherwise this would still be broken for the safari, right?